tests/scripts/all.sh

```diff
@@ -2100,6 +2106,15 @@ component_test_memsan_constant_flow_psa () {
     make test
 }
 
+support_test_valgrind_constant_flow () {
```

Comment:
This hack isn't going to work: the CI will complain about components that exists but aren't getting executed.

I would like to split `all.sh` into several parts: the logic, and multiple files containing components. Then we'll be able to have different sets of components for different circumstances. There's ongoing work on this for the sake of TF-PSA-Crypto.